dashboard/app/jobs_test.go

```diff
@@ -260,13 +262,14 @@ Tested on:
 
 commit:         11111111 kernel_commit_title1
 git tree:       repo1 branch1
+console output: %[4]v
```

Comment:
If this is the only diff, it looks good.

Reply:
Yes, nothing more. Actually absolutely no changes on the dashboard side.